### ISSUE TITLE
feat: Updated train.py.

### ISSUE THREAD

**Issue by dguevar1** [dguevar1/image_classifier#18]
This issue updates train.py to print out validation loss and validation accuracy.
